frontend/views/containers/proposals/ProposalVoteOptions.vue

```diff
@@ -0,0 +1,202 @@
+<template lang='pug'>
+.c-ctas
+  .buttons(v-if='!hadVoted || ephemeral.changingVote')
+    i18n.button.is-outlined.is-small.is-success(
+      tag='button'
+      @click='voteFor'
+      data-test='voteFor'
+    ) Vote yes
+
+    i18n.button.is-outlined.is-small.is-danger(
+      tag='button'
+      @click='voteAgainst'
+      data-test='voteAgainst'
+    ) Vote no
+
+  .c-cancel(v-else)
+    i18n.button.is-outlined.is-small(
+      tag='button'
+      v-if='ownProposal'
+      @click='cancelProposal'
+      data-test='cancelProposal'
+    ) Cancel Proposal
+    p.has-text-1(v-else data-test='voted')
+      | {{ L('You voted {voteStatus}', { voteStatus }) }}.
+      | &nbsp;
+      i18n.link(tag='button' @click='startChangingVote') Change vote.
+  .help.has-text-danger.c-error(v-if='ephemeral.errorMsg') {{ ephemeral.errorMsg }}
+</template>
+
+<script>
+import sbp from '~/shared/sbp.js'
+import L from '@view-utils/translations.js'
+import { VOTE_FOR, VOTE_AGAINST } from '@model/contracts/voting/rules.js'
+import { TYPE_MESSAGE } from '@model/contracts/mailbox.js'
+import { oneVoteToPass, buildInvitationUrl } from '@model/contracts/voting/proposals.js'
+import { generateInvites } from '@model/contracts/group.js'
+import { mapGetters } from 'vuex'
+
+export default {
+  name: 'Vote',
+  props: {
+    proposalHash: String
+  },
+  data () {
+    return {
+      ephemeral: {
+        changingVote: false
+      },
+      errorMsg: null
+    }
+  },
+  computed: {
+    ...mapGetters([
+      'currentGroupId',
+      'currentGroupState',
+      'groupSettings',
+      'currentUserIdentityContract'
+    ]),
+    proposal () {
+      return this.currentGroupState.proposals[this.proposalHash]
+    },
+    currentUsername () {
+      return this.currentUserIdentityContract.attributes.name
+    },
+    voteStatus () {
+      const humanStatus = {
+        [VOTE_FOR]: 'yes',
+        [VOTE_AGAINST]: 'no'
```

Comment:
heh. Do you see what's the issue here? :P

Reply:
fixed